examples/pytorch/eeg-gcnn/main.py

```diff
@@ -0,0 +1,240 @@
+import argparse
+import numpy as np
+import pandas as pd
+import torch
+import torch.nn as nn
+from sklearn.model_selection import train_test_split
+from joblib import load, dump
+from deep_EEGGraphConvNet import EEGGraphConvNet
+from EEGGraphDataset import EEGGraphDataset
+from dgl.dataloading import GraphDataLoader
+from torch.utils.data import WeightedRandomSampler
+from sklearn.metrics import roc_auc_score
+from sklearn.metrics import balanced_accuracy_score
+from sklearn import preprocessing
+
+
+if __name__ == "__main__":
+    # argparse commandline args
+    parser = argparse.ArgumentParser(description='Execute training pipeline on a given train/val subjects')
+    parser.add_argument('--num_feats', type=int, default=6, help='Number of features per node for the graph')
+    parser.add_argument('--num_nodes', type=int, default=8, help='Number of nodes in the graph')
+    parser.add_argument('--gpu_idx', type=int, default=0,
+                        help='index of GPU device that should be used for this run, defaults to 0.')
+    parser.add_argument('--num_epochs', type=int, default=40, help='Number of epochs used to train')
+    parser.add_argument('--exp_name', type=str, default='default', help='Name for the test.')
+    parser.add_argument('--batch_size', type=int, default=512, help='Batch Size. Default is 512.')
+    args = parser.parse_args()
```

Comment:
I guess we can have an argument to control the use of the deep variant or shallow variant.